### PR TITLE
NEWSOURCE-176: Archetype implementation

### PR DESCRIPTION
**Pull request by ElroTheOnk**
Adds the following archetypes: Nirmathi Irregular, Primalist, Razmiran Priest, Shadowcaster, Spire Defender, Sword of Valor, Tempest Druid, Vampire Hunter, Winter Witch

Untested but no load errors. There are a couple of archetype ability switchouts that depend on Core work, held pending FACT project.

Still TODO on this source: fleshwarped template, magic schools abilities & testing

### CODE REVIEW

File: pathfinder/paizo/campaign_setting/inner_sea_magic/ism_abilities_class.lst [hunk 1]
```diff
@@ -227,17 +269,49 @@ Black Blood Spray	KEY:Oracle ~ Black Blood Spray	CATEGORY:Special Ability	TYPE:O
...
+Enhanced Varisian Tattoo	KEY:Enhanced Varisian Tattoo ~ Tattooed Sorcerer	NAMEISPI:YES	CATEGORY:Special Ability	TYPE:SorcererClassFeatures.SpecialQuality.Supernatural.SorcererBloodlinePowerLvl9																		DESCISPI:YES	DESC:The tattooed sorcerer can pick any one spell she knows for which she has a Varisian Tattoo feat. This spell must be one that lacks focus components and costly material components. She can now use that spell as a spell-like ability once per day. This spell-like ability is not enhanced by her Varisian Tattoo, but it functions at +2 caster levels above her sorcerer caster level. Whenever she gains a bloodline power at a later date, she may change this spell-like ability to another qualifying spell. This ability replaces the 9th-level bloodline power.																																																																																																																											SOURCEPAGE:p.41
 # Varisian Tattoo Abilities (for Tattooed Sorcerer without Spell Focus feat)
```
Comment (LegacyKing) [inline pathfinder/paizo/campaign_setting/inner_sea_magic/ism_abilities_class.lst]: Grouping is off... Tattooed Sorcerer ~ xxx

File: pathfinder/paizo/campaign_setting/inner_sea_magic/ism_classes.lst [hunk 2]
```diff
@@ -12,6 +12,9 @@ CLASS:Eidolon (Fey)	PRERACETYPE:Eidolon
 # Class Name		Skill Pts/Lvl	Add INT to Skill?	Class Skill
 CLASS:Eidolon (Fey)	STARTSKILLPTS:6	MODTOSKILLS:YES	CSKILL:Acrobatics|Bluff|Climb|TYPE=Craft|Diplomacy|Disguise|Escape Artist|Fly|Knowledge (Geography)|Knowledge (Nature)|Perception|TYPE=Perform|Sense Motive|Sleight of Hand|Stealth|Swim|Use Magic Device
 
+###Block: Witch ~ Winter Witch
+CLASS:Witch.MOD	PROHIBITSPELL:DESCRIPTOR.Fire	PREABILITY:1,CATEGORY=Special Ability,Winter Witch ~ Ice Magic
+
```
Comment (LegacyKing) [inline pathfinder/paizo/campaign_setting/inner_sea_magic/ism_classes.lst]: What is the purpose of this change? This seems to say I can no longer play a Witch unless I have "Winter Witch ~ Ice Magic"

File: pathfinder/paizo/campaign_setting/inner_sea_magic/ism_abilities_class.lst (hunk 1 above)
Comment (ElroTheOnk) [inline pathfinder/paizo/campaign_setting/inner_sea_magic/ism_abilities_class.lst]: This one was already in - didn't modify it. Will fix to standard form & resubmit.

File: pathfinder/paizo/campaign_setting/inner_sea_magic/ism_classes.lst (hunk 2 above)
Comment (ElroTheOnk) [inline pathfinder/paizo/campaign_setting/inner_sea_magic/ism_classes.lst]: PREABILITY should only apply to the PROHIBITSPELL of course. Will fix & resubmit.

**Comment by ElroTheOnk** [comment]
Updated PR in response to Andrew's comments

**Pull request merged**